python/src/ngraph/ops.py

```diff
@@ -2588,3 +2588,83 @@ def reverse_sequence(input, seq_lengths, batch_axis, seq_axis, name=None):
     """
     return _get_node_factory().create('ReverseSequence', [input, as_node(seq_lengths)],
                                       {'batch_axis': batch_axis, 'seq_axis': seq_axis})
+
+
+@nameable_op
+def bucketize(data, buckets, output_type='i64', with_right_bound=True):
```

Comment:
To be consistent with the rest of pyAPI ops and `@namable_op` decorator, 
please  add  optional arg `name=None,  # type: str`.